cogboard-webapp/src/hooks/index.js

```diff
@@ -7,11 +7,10 @@ export const useToggle = (initialState = false) => {
   const [isOpened, setOpened] = useState(initialState);
   const [expanded, setExpanded] = useState(false);
 
-  const handleOpen = () => setOpened(true);
-  const handleClose = () => setOpened(false);
-  const handleToggle = () => setExpanded(!expanded);
+  const handleToggle = () => setOpened(!isOpened);
+  const handleExpand = () => setExpanded(!expanded);
 
-  return [isOpened, handleOpen, handleClose, handleToggle, expanded];
+  return [isOpened, handleToggle, handleExpand, expanded];
```

Comment:
My code snippet what about not touching the existing handlers and just adding handleToggle method which sets the existing state based on the previous one. Please do not make such a changes, because a lot of components use this piece of state functionality. You can do it like refactoring the return to an object, so you can name import from it when you need particular piece. Then if you maintain many toggled states in the component, you can call useToggle many times instead on keeping the same false to true state in the hook. When you finish just make sure that other code using this hook works fine. Refactor will be needed.